### ISSUE TITLE
Relative paths are not considered when option "localeExtension" false

### ISSUE THREAD

**Issue by Akiyamka** [felixzapata/gulp-pug-i18n#5]
Example file structure:
```
gulpfile.js
src
  ├─ pages
  ├─ index.html
  ├─ about.html
  └─ blog
       ├─ index.html
       └─ article.html
```

After the task:

```
gulp.task('html', function () {
  var options = {
    i18n: {
      locales: 'locales/en_US.json',
      localeExtension: false,
      dest: 'translations/',
    },
    client: false,
    pretty: false
  };

  return gulp.src('src/pages/**/*.pug')
    .pipe(pugI18n(options))
    .pipe(gulp.dest(options.i18n.dest))
```

Will look like:

```
translations
  └─ en_US
        ├─ index.html
        ├─ about.html
        ├─ index1.html
        └─ article.html
```

Expected behavior: that the structure should be preserved, and result look like:

```
translations
└─ en_US
       ├─ pages
       ├─ index.html
       ├─ about.html
       └─ blog
             ├─ index.html
             └─ article.html
```

**Comment by felixzapata** [comment]
I will take a look ASAP.

**Comment by felixzapata** [comment]
@Akiyamka if you have the fix, please make a pull request please

**Comment by Akiyamka** [comment]
Already do. Not sure if this is the best way to fix the problem, but at least it works

**Comment by felixzapata** [comment]
I will take a look to the pull request. I was working in a similar solution.

**Comment by felixzapata** [comment]
Hi @Akiyamka , thanks for the pull request. I've just published the version 1.0.3.
